src/commands/BeautifyCommand.ts

```diff
@@ -0,0 +1,139 @@
+import { BeautifyData } from "unibeautify";
+import * as cosmiconfig from "cosmiconfig";
+import * as fs from "fs";
+
+import {
+  setupUnibeautify,
+} from "../utils";
+import { BaseCommand } from "./BaseCommand";
+
+export class BeautifyCommand extends BaseCommand {
+  protected stdin: NodeJS.ReadStream = process.stdin;
+
+  public beautify(programArgs: IArgs): Promise<string> {
+    const { language, filePath } = programArgs;
+    return setupUnibeautify().then(unibeautify => {
+      if (!language) {
+        const error = new Error("A language is required.");
+        this.writeError(error.message);
+        this.exitCode = 1;
+        return Promise.reject(error);
```

Comment:
Should we maybe have a `handleError` method of sorts that accepts a Error and exit code to reduce duplicate code?

```typescript
protected handleError(error: Error, exitCode: number): Promise<never> {
    this.writeError(error.message);
    this.exitCode = exitCode;
    return Promise.reject(error);
}
```

Reply:
I tried this, however, there is not explicit type for `Promise.reject` so it ended up confusing TypeScript and making the return type worse. 😞

Reply:
I think this can be refactored later. Will have to think a little more about it and test different ideas.

Reply:
It's not `Promise<never>`?  I just tried it and it worked fine.